client/src/pages/private/Admin.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Page, Button } from '../../components/generic';
+import store from 'store';
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default () => {
+
+  const handleSubmit = async () => {
+
+    const response = await fetch('/api/v1/admin', {
+      headers: {
+        'Accept': 'application/json',
+        'Content-type': 'application/json',
+        'Authorization': `Bearer ${store.get('TOKEN')}`,
+      },
+      method: 'GET',
+    });
+
+    if (response.ok) {
+      const { result } = await response.json();
+      console.log(result);
```

Comment:
Should remove this now or make a note somewhere to make sure we come back to this before EOW :)

Reply:
This component no longer exists in my next PR that is a superset of this one

Reply:
Awesome!